datalad/plugin/addurls.py

```diff
@@ -556,30 +556,24 @@ def add_meta(rows):
     """
     from unittest.mock import patch
 
+    # OPT: group by dataset first so to not patch/unpatch always_commit
+    # per each file of which we could have thousands
+    from collections import defaultdict
```

Comment:
Fine as is, but we use collections widely enough that delaying this import doesn't save us anything.

Reply:
Thanks for the review! Forgot to move it up ;)